clustermesh-apiserver/clustermesh/k8s/namespace.go

```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package k8s
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/spf13/pflag"
+
+	"github.com/cilium/cilium/pkg/annotation"
+	"github.com/cilium/cilium/pkg/k8s/resource"
+	slim_corev1 "github.com/cilium/cilium/pkg/k8s/slim/k8s/api/core/v1"
+)
+
+type NamespaceConfig struct {
+	EnableDefaultGlobalNamespace bool
+}
+
+func (def NamespaceConfig) Flags(flags *pflag.FlagSet) {
+	flags.Bool("enable-default-global-namespace", def.EnableDefaultGlobalNamespace, "Enables the global namespace feature")
```

Comment:
clustermesh flags usually starts with `clustermesh-, maybe this flag could be named `clustermesh-default-global-namespace` then?

Reply:
Changed to `clustermesh-default-global-namespace`.